index.html

```diff
@@ -83,6 +83,10 @@ <h2>Join the Club</h2>
           <td><a href="https://github.com/natalie-lang/natalie">Natalie</a></td>
           <td>a Ruby compiler, for fun</td>
         </tr>
+        <tr>
+          <td><a href="https://github.com/SerenityOS/serenity">SerenityOS</a></td>
+          <td>a operating system from scratch, for fun</td>
```

Comment:
@seven1m Where does the restriction 

> The first word must be a lowercase "a".

come from? Perhaps that consumer should be updated to recognize `an` as well, for better grammaticallness :)

Reply:
Yeah, that should be a/an. 😄

Reply:
I fixed it, and update the comment and PR template directions. Thanks.

Reply:
just kidding... looks like Tim beat me to it.